src/app/availability/availability-table/availability-table.component.html

```diff
@@ -0,0 +1,67 @@
+﻿<p-toast></p-toast>
+<app-menubar></app-menubar>
+
+<p-card header="Podawanie dostępności">
+  <ng-template [ngIf]="declarableSprint">
+
+    <div *ngIf="availability">
+      <div class="ui-g">
+        <div class="ui-g-4">
+          <h5>Data rozpoczęcia:</h5>
+        </div>
+        <div class="ui-g-4">
+          <span>{{ declarableSprint.durationPeriod.start | date: 'dd-MM-yyyy HH:mm' }}</span>
+        </div>
+      </div>
+      <div class="ui-g">
+        <div class="ui-g-4">
+          <h5>Data zakończenia:</h5>
+        </div>
+        <div class="ui-g-4">
+          <span>{{ declarableSprint.durationPeriod.end | date: 'dd-MM-yyyy HH:mm' }}</span>
+        </div>
+      </div>
+      <div class="ui-g">
+        <div class="ui-g-4" style="font-style: italic">
+          <h6>Możliwość deklaracji dostępności
+            mija {{ declarableSprint.enrollmentPeriod.end | date: 'dd-MM-yyyy HH:mm' }}</h6>
+        </div>
+      </div>
+      <div class="ui-g">
+        <div class="ui-g-4">
+          <h5>Podaj łączną liczbę godzin jakie przepracujesz w tym sprincie:</h5>
+        </div>
+        <div class="ui-g-4">
+          <input type="number" [(ngModel)]="availability.timeAvailable" placeholder="Deklarowane"/>
```

Comment:
ujemne wartości

Reply:
Ale to na backendzie też, nie?

Reply:
Bo `Duration.ofMinutes` najwyraźniej nie ma nic przeciwko ujemnym liczbom